subsys/bluetooth/mesh/gen_ponoff_srv.c

```diff
@@ -258,8 +258,15 @@ static void scene_recall_complete(struct bt_mesh_model *model)
 	(void)bt_mesh_onoff_srv_pub(&srv->onoff, NULL, &status);
 }
 
+/*  MeshMDL1.0.1, section 5.1.3.1.1:
+ *  If a model is extending another model, the extending model shall determine
+ *  the Stored with Scene behavior of that model.
+ *
+ *  Use Setup Model to handle Scene Store/Recall as it is not extended
+ *  by other models.
+ */
 BT_MESH_SCENE_ENTRY_SIG(ponoff) = {
-	.id.sig = BT_MESH_MODEL_ID_GEN_POWER_ONOFF_SRV,
+	.id.sig = BT_MESH_MODEL_ID_GEN_POWER_ONOFF_SETUP_SRV,
```

Comment:
I initially had a comment here about this breaking the scene loading extended models mechanism, but I was wrong. Neat solution :+1: